### ISSUE TITLE
Broken direct links in README.md

### ISSUE THREAD

**Issue by jmfayard** [parse-community/parse-server#3692]
Hello, the `README` is hard to follow because it contains a lot of direct links that are not direct anymore but instead redirect to a generic page like https://github.com/parse-community/parse-server/wiki and http://parseplatform.org/

I have identified them below section by section 

```markdown
Saving your first object

- [REST API](https://parse.com/docs/rest/guide)


Documentation

The [Parse Server guide](https://github.com/ParsePlatform/parse-server/wiki/Parse-Server-Guide) is a good place to get started.

the [Development guide](https://github.com/ParsePlatform/parse-server/wiki/Development-Guide) will help you get set up.


Migrating an Existing Parse App

Learn more in the [Migration guide](https://parse.com/migration).


Basic options

Configuration options for APNS and GCM push. See the [Push Notifications wiki entry](https://github.com/ParsePlatform/parse-server/wiki/Push).


Advanced options

`auth` - Used to configure support for [3rd party authentication](https://github.com/ParsePlatform/parse-server/wiki/OAuth--and-Custom-Authentication).


Configuring File Adapters

`GridStoreAdapter` is used by default and requires no setup, but if you're interested in using S3 or Google Cloud Storage, additional configuration information is available in the [Parse Server wiki](https://github.com/ParsePlatform/parse-server/wiki/Configuring-File-Adapters).

additional configuration information is available in the [Parse Server wiki](https://github.com/ParsePlatform/parse-server/wiki/Configuring-File-Adapters).


Support

You've met the [prerequisites](https://github.com/ParsePlatform/parse-server/wiki/Parse-Server-Guide#prerequisites).
```

**Comment by jmfayard** [comment]
The same link `http://parseplatform.github.io/docs/parse-server/guide/#push-notifications` is also broken in the README of https://github.com/parse-server-modules/parse-server-push-adapter

**Comment by jmfayard** [comment]
That link is also broken on the front page of http://parseplatform.org/

![image](https://cloud.githubusercontent.com/assets/459464/24746155/60c50a28-1ab8-11e7-8d10-3be8c6e1a790.png)

**Comment by flovilmart** [comment]
@jmfayard Thanks! I'll Try to fix it all up!

**Comment by flovilmart** [comment]
Thanks for reporting @jmfayard! I believe this should be good now.

**Comment by jmfayard** [comment]
@flovilmart works except for the https links for things hosted on githug. 

I added comments in the commit

![image](https://cloud.githubusercontent.com/assets/459464/24793967/3a690bb4-1b84-11e7-95a1-a11ac66a9006.png)

**Comment by flovilmart** [comment]
Ah right... need to move those to http! Thanks!
